GetIntoTeachingApi/Models/Crm/Validators/CandidateValidator.cs

```diff
@@ -59,6 +59,7 @@ public CandidateValidator(IStore store, IDateTimeProvider dateTime)
                 .SetValidator(new PickListItemIdValidator<Candidate>("contact", "dfe_ittyear", store))
                 .Unless(candidate => candidate.InitialTeacherTrainingYearId == null);
             RuleFor(candidate => candidate.ChannelId)
+                .NotNull()
```

Comment:
I think there's a mistake on line 64 that if we correct should mean it doesn't need this `NotNull` -- it should be `Unless(c => c.Id == null)` but if we want it to be required we can just remove the `Unless` clause and have only the `SetValidator` which will enforce it to be one of the valid values and not `null`.

Reply:
This is to satisfy this test https://github.com/DFE-Digital/get-into-teaching-api/blob/bab56c6b81ecf94ef8c835d7c86d22bf437849b3/GetIntoTeachingApiTests/Models/Crm/Validators/CandidateValidatorTests.cs#L495